src/js/modules/ps_currencyselector.ts

```diff
@@ -7,6 +7,11 @@ const initCurrencySelector = () => {
   const {Theme} = window;
   const {currencySelector: CurrencySelectorMap} = Theme.selectors;
   const currencySelector = document.querySelector<HTMLElement>(CurrencySelectorMap.currencySelector);
+  const currencyMobileSelector = document.querySelector<HTMLElement>(CurrencySelectorMap.currencyMobileSelector);
+
+  if (!currencySelector) {
+    currencyMobileSelector?.remove();
```

Comment:
This will surely cause content jump

Reply:
@nizelg it's hidden under off-canvas if I'm not mistaken, shouldn't cause CLS